TWLight/templates/home.html

```diff
@@ -23,7 +25,7 @@
         </a>
       {% else %}
         <a href="{% url 'oauth_login' %}" class="btn btn-primary btn-block">
-          <span class="glyphicon glyphicon-off"></span>&nbsp;
+          <span class="glyphicon glyphicon-off"></span>&nbsp
```

Comment:
why this change? `&nbsp;` is a valid html entity, but `&nbsp` isn't. Was it rendering incorrectly?

Reply:
Once this is resolved (whether a change is required or not), feel free to merge.

Reply:
Oh, weird. PyCharm implied this was an end-of-line space, I think. I'll just fix this on Github and merge now.